### PR TITLE
Popup page

### PR DESCRIPTION
**Pull request by 2004-okari**
In this project:

1. When the user clicks the button to check project details, the popup with details about the project appears.
2. When the user clicks the close (X) button, the popup disappears.
3. Used a JavaScript array to store all of the information for all projects.
4. Updated the main page so that the projects section is created dynamically using the information stored in that JavaScript object. 
5.  When a user is looking a the popup card there is no body scroll to prevent going to unnecessary pages.

### CODE REVIEW

**Review by LionRouge1 — COMMENTED** [review]
Hi @2004-okari, 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that was supposed to be a pair-programming activity but you're the only one who contributed to the project. Kindly work with your partner to submit this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: pop.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                </div>
+            </div>
+            <div class="pop-2">
+                <img src="${projects[i - 1].featuredImage}" class="image-preview">
+                <div class="pop-3">
+                    <p class="card-text">${projects[i - 1].description}</p>
+                    <div class="card-btns">
+                        <button type="button" class="card-btn-1" onclick="window.location.href='${projects[i - 1].linkToLiveVersion}';">See live<img src="images/Icon.svg" alt="icons" class="btn-icons"></button>
+                        <button type="button" class="card-btn-2" onclick="window.location.href='${projects[i - 1].linkToSource}';">See source<img src="images/Vector.svg" alt="icons" class="btn-icons"></button>
+                    </div>
```
Comment (danmainah) [inline pop.js:134]: - [ ] @team great wrok ading the projects dynamically,however there are some design differences with the design, popup window background is not blurred in both desktop/mobile view, also the positioning of the buttons in the desktop view is incorrect and the title should be in one line, check the comparison below:


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image| image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @2004-okari and @NalianAchei2020 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Added the project section dynamically using index.js file, and ensured the popup are displayed correctly.
✔️  Ensured the popup window design matches the Figma design given.


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: pop.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
+//  Javascript popup
+
+const projects = [
+  {
+    name: 'Proffesional Art<br>Printing Data',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    featuredImage: 'images/Snapshoot Portfolio.svg',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.<br> It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    linkToLiveVersion: 'https://2004-okari.github.io/MY-PORTFOLIO/',
+    linkToSource: 'https://github.com/2004-okari/MY-PORTFOLIO',
```
Comment (Shedrack-Sunday) [inline pop.js]: - [ ] Nice efforts so far. However, you still have to adjust the popup based on the changes asked by the previous reviewer.
- The background content is still not burry when the popup is clicked. The whole focus should be on the popup.

### Sample code to make it burry:
```
.popup-background {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background-color: rgba(0, 0, 0, 0.7); /* Semi-transparent black overlay */
  backdrop-filter: blur(5px); /* Apply the blur effect */
  z-index: 9999; /* Ensure the background appears above other elements */
}
```
- The `buttons` are meant to be at the bottom. Proper use of a grid can accomplish this to place the items 

### Your view:
image

### Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello Team   👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice work with the setup✔️  
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirement



_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by V-Blaze — APPROVED** [review]
Hi @2004-okari  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Pop-up works great💯
-  Project section generated dynamically👍
-  Correct Github flow🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**